### PR TITLE
[Docs] Adding Zicond in user and requirement specs

### PR DESCRIPTION
**Pull request by fatimasaleem**
Closes https://github.com/openhwgroup/cva6/issues/1270 & https://github.com/openhwgroup/cva6/issues/1271

### CODE REVIEW

**Comment by AyoubJalali** [comment]
> @AyoubJalali can you approve the PR? To check whether the instructions description is aligned on the original specification

LGTM!! Thanks @fatimasaleem

**Comment by AyoubJalali** [comment]
@fatimasaleem hello could I find, any document with the encoding 32bits of these instructions (e.g opcode, func ...)

File: docs/01_cva6_user/Programmer_View.rst [hunk 1]
```diff
@@ -39,6 +39,7 @@ RISC-V Extensions
    "Zicount - Performance Counters",                                    "Yes","✓","✓"
    "Zicsr - Control and Status Register Instructions",                  "No","✓","✓"
    "Zifencei - Instruction-Fetch Fence",                                "No","✓","✓"
+   "Zicond - Integer Conditional Operations",                           "Yes","✓","✓"
```
Comment (jquevremont) [inline docs/01_cva6_user/Programmer_View.rst:42]: Can you reword "Zicond - Integer Conditional Operations (Ratification pending)"?

**Review by jquevremont — CHANGES_REQUESTED** [review]

File: docs/02_cva6_requirements/cva6_requirements_specification.rst [hunk 2]
```diff
@@ -311,6 +311,11 @@ independent requirements.
 |                                   | **Zbc** and **Zbs** extensions    |
 |                                   | (bit manipulation), version 1.0.  |
 +-----------------------------------+-----------------------------------+
+| ISA-130                           | CVA6 should support as an         |
+|                                   | **option** the **Zicond**         |
+|                                   | extension, version 1.0.           |
```
Comment (jquevremont) [inline docs/02_cva6_requirements/cva6_requirements_specification.rst:316]: I thought that Zicond was not ratified yet. May I suggest to reword "CVA6 should support as an **option** the **Zicond** extension (ratification pending).

**Review by jquevremont — CHANGES_REQUESTED** [review]

**Comment by fatimasaleem** [comment]
> @fatimasaleem hello could I find, any document with the encoding 32bits of these instructions (e.g opcode, func ...)

@AyoubJalali [here](https://github.com/riscv/riscv-zicond/releases/tag/v1.0-rc4) is a link to the Zicond specifications.

**Pull request merged**